src/raygun.breadcrumbs.js

```diff
@@ -0,0 +1,296 @@
+/*
+ * raygun4js
+ * https://github.com/MindscapeHQ/raygun4js
+ *
+ * Copyright (c) 2017 MindscapeHQ
+ * Licensed under the MIT license.
+ */
+/* globals console */
+
+var raygunBreadcrumbsFactory = function(window, $, Raygun) {
```

Comment:
Ahh I see. I'd be tempted to move it into one but probably better to maintain the current convention.